crypto/evp/p_lib.c

```diff
@@ -408,6 +363,7 @@ EVP_PKEY *EVP_PKEY_new_raw_public_key(int type, ENGINE *e,
 int EVP_PKEY_get_raw_private_key(const EVP_PKEY *pkey, unsigned char *priv,
                                  size_t *len)
 {
+    /* TODO(3.0) Do we need to do anything about provider side keys? */
```

Comment:
looks like we may be able to call my new API's for provider side keys??

Reply:
Other PR